apps/tasks/models.py

```diff
@@ -1,3 +1,44 @@
-# from django.db import models
+from django.db import models
+# from tasks.models import Users
+from enumchoicefield import ChoiceEnum, EnumChoiceField
+import uuid
 
-# Create your models here.
+
+class Status(ChoiceEnum):
+    BACKLOG = 'Backlog',
+    IN_PROGRESS = 'InProgress',
+    DONE = 'Done',
+
+
+class Priority(ChoiceEnum):
+    LOW = 'Low',
+    MEDIUM = 'Medium',
+    HIGH = 'High',
+    CRITICAL = 'Critical',
+
+
+class Task(models.Model):
+    id = models.UUIDField(default=uuid.uuid4, unique=True,
+                          primary_key=True, editable=False)
+    title = models.CharField(max_length=200)
+    created_date = models.DateTimeField(auto_now_add=True)
+    # created_by = models.ForeignKey(User)
+    # assignee = models.ForeignKey(User)
```

Comment:
Since these lines are future to be uncommented, I refer to them as they were part of the code.
Maybe add on_delete=models.RESTRICT, this adds some protection to our code and prevent a task to be asignee-less.